Resources/translations/validators.de.xliff

```diff
@@ -3,10 +3,16 @@
     <file source-language="en" target-language="de" datatype="plaintext" original="file.ext">
         <header/>
         <body>
-            <trans-unit id="26" resname="error.uniq_url">
+            <trans-unit id="error.uniq_url" resname="error.uniq_url">
                 <source>error.uniq_url</source>
                 <target>Die URL '%url%' ist bereits mit einer anderen Seite verknüpft</target>
             </trans-unit>
+            <trans-unit id="error.uniq_url.parent_unselect" resname="error.uniq_url.parent_unselect">
+                <source>error.uniq_url.parent_unselect</source>
+                <target>Die Basis-URL '/' ist bereits mit einer anderen Seite verknüpft.
+                Bitte wählen Sie eine übergeordnet Seite
```

Comment:
> May I add them in all files ?

Sure you can if you want. Just make sur you add them at then end of sentences. For instance "Bad credentials" is not a sentence.